.github/workflows/release.yaml

```diff
@@ -120,19 +159,60 @@ jobs:
           echo "name:        ${{ steps.import_gpg.outputs.name }}"
           echo "email:       ${{ steps.import_gpg.outputs.email }}"
 
-      - name: Build & publish release artifacts
-        run: make release
+      - name: Build for Linux
+        run: make build-linux
+
+      - name: Create Linux release assets
+        run: make package-linux
         env:
-          GITHUB_TOKEN: ${{ secrets.ANCHORE_GIT_READ_TOKEN }}
           GPG_PRIVATE_KEY: ${{ secrets.SIGNING_GPG_PRIVATE_KEY }}
           PASSPHRASE: ${{ secrets.SIGNING_GPG_PASSPHRASE }}
-          SIGNING_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
-          AWS_ACCESS_KEY_ID: ${{ secrets.TOOLBOX_AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.TOOLBOX_AWS_SECRET_ACCESS_KEY }}
-          APPLE_DEVELOPER_ID_CERT: ${{ secrets.APPLE_DEVELOPER_ID_CERT }} # Used during macOS code signing.
-          APPLE_DEVELOPER_ID_CERT_PASS: ${{ secrets.APPLE_DEVELOPER_ID_CERT_PASS }} # Used during macOS code signing.
-          AC_USERNAME: ${{ secrets.ENG_CI_APPLE_ID }} # Used during macOS notarization.
-          AC_PASSWORD: ${{ secrets.ENG_CI_APPLE_ID_PASS }} # Used during macOS notarization.
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: Linux-artifacts
+          path: |
+            ./dist/*_linux_*.*
+            ./dist/*_checksums.*
+
+  draft-release:
+    needs: [ release-assets-mac, release-assets-linux ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/download-artifact@v2 # Downloads all artifacts
+
+      - name: Generate changelog
+        run: make changelog-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+
+      - name: Create draft release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          body_path: "./CHANGELOG.md"
+          draft: true
+          prerelease: false
+
+      - name: Upload release assets
+        run: | # Solution found at https://github.com/actions/upload-release-asset/issues/28#issuecomment-617208601 after seeing that the native "actions/upload-release-asset" might not be actively maintained.
+          set -eux
+          assets=()
+          for asset in ./Linux-artifacts/*; do
+            assets+=("-a" "$asset")
+          done
+          for asset in ./macOS-artifacts/*; do
+            assets+=("-a" "$asset")
+          done
+          tag_name="${GITHUB_REF##*/}"
+          hub release edit -m "" "${assets[@]}" "$tag_name"
```

Comment:
I guess `hub` is just available inside an action right?

Reply:
Exactly (see here for what's installed natively on GHA runners for [Linux](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md#cli-tools) and [macOS](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#utilities))